user-guide/Standard_Apps/IDP/Installing_DataMiner_IDP.md

```diff
@@ -4,6 +4,63 @@ uid: Installing_DataMiner_IDP
 
 # Installing DataMiner IDP
 
+IDP can be found on the [Catalog](https://catalog.dataminer.services/details/package/XXXX).
+
+> [!NOTE]
+> The minimum supported DataMiner version is **10.2.0 CU0**.
```

Comment:
Minimum version will be 10.3.0 CU0 instead.

Reply:
Forgot to commit the change